src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -10533,6 +10532,20 @@ void Unit::ModSpellCastTime(SpellInfo const* spellInfo, int32 & castTime, Spell*
         castTime = int32(float(castTime) * m_modAttackSpeedPct[RANGED_ATTACK]);
     else if (IsPartOfSkillLine(SKILL_COOKING, spellInfo->Id) && HasAura(67556)) // cooking with Chef Hat.
         castTime = 500;
+
+    // Give us another option to change spell speeds
+    switch (spellInfo->Id)
+    {
+        case 99172:
+        case 99235:
+        case 99236:
+            if (Map* map = GetMap())
+                if (map->IsHeroic())
+                    castTime = spellInfo->CalcCastTime() - 500;
```

Comment:
Could use some input about this hack, are there any other alternatives to changing spell cast time?

Reply:
Looks like they removed this db2 file from the client, uhhh... this really sucks.

Reply:
Unless they moved this information to another file then we probably need to extract the information contained in the last revision of that file and hardcode the fixes or something.

The spell misc corrections provided from the file is to important to lose.

Reply:
yes, SpellMiscDifficulty.db2 was dropped, it was a bit redundant, now all informations are in SpellMisc.db2

long time ago, i was working on this and stopped because of doing cherry picks from 335 which will cause many trouble with this

for now, don't care about it in this pr